taichi/transforms/lower_ast.cpp

```diff
@@ -391,7 +391,9 @@ class LowerAST : public IRVisitor {
           expr->stmt);
     } else if (dest.is<IndexExpression>()) {
       auto ix = dest.cast<IndexExpression>();
+
```

Comment:
Are these empty lines added by intention?

Reply:
This is weird. Let me remove this